### ISSUE TITLE
As the Designer, I need to create internal commands to control the header and footer content so I can allow a User to override my defaults.

### ISSUE THREAD

**Issue by Traap** [Traap/tlc-article#10]
This issue must be done prior to #9.

Prepare tlc-article for extension by using:
1) \newcommand*
2) \tlc@ 
where appropriate.

**Comment by Traap** [comment]
I refactored tlc-article.cls based on reading LaTeX packages and classes documentation.

- hyperref is now the last package required.
- internal commands use \newcommand* syntax.
- internal command names use tlc@ syntax.

[GettingStarted.pdf](https://github.com/Traap/tlc-article/files/690611/GettingStarted.pdf)

**Comment by Traap** [comment]
Merged into master.

**Comment by Traap** [comment]
This issues was completed with PR #11
